src/Microsoft.AspNet.Http.Abstractions/Extensions/UseMiddlewareExtensions.cs

```diff
@@ -166,7 +192,12 @@ private static object GetService(IServiceProvider sp, Type type, Type middleware
             var service = sp.GetService(type);
             if (service == null)
             {
-                throw new InvalidOperationException(Resources.FormatException_InvokeMiddlewareNoService(type, middleware));
+                throw new InvalidOperationException(
+                    string.Format(
+                        Resources.Exception_InvokeMiddlewareNoService,
+                        type,
+                        middleware
+                    ));
```

Comment:
@Tratcher was correct about why I changed those, but now that I'm not touching HttpAbstractions It's all reverted.